src/main/java/Main.java

```diff
@@ -5,18 +5,15 @@
 public class Main {
 
     public static void main(String[] args) throws IOException {
-
+        // Employee employee = new Employee("Adam", "Janiczek", "87101404616", "IsdfadsfasT", 50000.00);
         File file = new File("employees.csv");
-
         if (file.exists()) {
+            Employee[] employees = FileUtils.read(file);
+            EmployeeStats employeeStats = new EmployeeStats(employees);
             File stats = new File("stats.txt");
-            stats.createNewFile();
-
-//            FileWriter fileWriter = new FileWriter(stats);
-//            fileWriter.write("Średnia wypłata: 5000");
-//
-//            fileWriter.close();
+            FileWriter fileWriter = new FileWriter(stats);
+            fileWriter.write(employeeStats.toString());
+            fileWriter.close();
```

Comment:
Co prawda zamykasz ten plik później, ale w przypadku błędu może pozostać niezamknięty. Najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY